revisions-extended/src/plugins/editor-modifications/update-button-modifier/index.js

```diff
@@ -34,14 +40,34 @@ const UpdateButtonModifier = () => {
 	} = usePost();
 
 	const _savePost = async () => {
+		const isFutureRevision = changingToScheduled();
+		const noticeDispatch = dispatch( 'core/notices' );
+
+		// We currently don't properly support 'pending' revisions,
+		// Alert user they should select a date in the future
+		if ( ! isFutureRevision ) {
+			noticeDispatch.createWarningNotice(
+				__(
+					'We currently only support updates with future publish dates. Please select a date in the future.'
+				),
```

Comment:
Needs the `revisions-extended` text domain as a second argument.

Reply:
Curious about text domains... Why doesn't Gutenberg have them?  Also, I **very much** dislike needing to code the text domain (the linter doesn't allow variables to be passed in the second position.).

Reply:
Translation strings in _Core_ don't use a textdomain, because it's the default value. Translation strings in the Gutenberg plugin still use a textdomain, I think...

Yah, it's kind of a pain that you can't use variables in the translation functions. I think it's because the strings are extracted from the files statically, so there's no variable resolution during the process.